src/main/ts/core/BranchLogic.ts

```diff
@@ -37,7 +38,12 @@ export const enum BranchState {
   ReleaseCandidate = 'releaseCandidate'
 }
 
-export interface Module {
+interface ModuleChangelog {
+  readonly changelogFile: string;
+  readonly changelogFormat: string;
```

Comment:
Would it be worth having an explicit set of valid strings here?
```suggestion
  readonly changelogFormat: 'keepachangelog' | 'none';
```

Reply:
Ahh yep I meant to come back and do this but forgot. At one stage this wouldn't have worked but forgot to change if after I made changes so this would work. Anyways, it's been fixed now thanks.